build.gradle

```
@@ -444,6 +444,10 @@ def createJsTestTask(String... subprojectNames) {
             from compileKotlin2Js.destinationDir
 
             prefixedProject('verbs-internal-js').afterEvaluate {
+                configurations.testRuntimeClasspath.allDependencies.withType(ProjectDependency).each {
```

Can you please add a TODO for me, I guess this should no longer be necessary once we switched to the new Kotlin MPP plugin
```suggestion
                // TODO 0.16.0 or 0.17.0, check if still required with the new Kotlin MPP plugin
                configurations.testRuntimeClasspath.allDependencies.withType(ProjectDependency).each {
```